vortex-file/src/read/mod.rs

```diff
@@ -48,7 +48,13 @@ impl Scan {
         Self { expr: None }
     }
 
-    pub fn new(expr: Option<ExprRef>) -> Self {
+    pub fn new(expr: ExprRef) -> Self {
```

Comment:
while we're here: AFAICT we don't need to expose Scan on the public API at all

Reply:
always a win

Reply:
actually we do because `Layout` is a public trait that we probably want to keep that way to allow people to extend Vortex

Reply:
ah missed that